README.md

```diff
@@ -10,77 +10,4 @@ comments.
-  http://www.oracle.com/technetwork/oracle-labs/program-languages/downloads
-* Unpack the downloaded `graalvm_*.tar.gz` into `simplelanguage/graalvm` (note: no version in folder name).
-* Set environment variable `JAVA_HOME` to the path of the `graalvm` folder (note: mac users should set `JAVA_HOME` to `/path/to/graalvm/Contents/Home`)
-* Verify that the file `$JAVA_HOME/bin/java` exists and is executable
-* Execute `mvn package`
-
-## IDE Setup
-
-### Eclipse
-* Tested with Eclipse Neon.2 Release (4.6.2)
-* Open Eclipse with a new workspace
-* Install `m2e` and `m2e-apt` plugins from the Eclipse marketplace (Help -> Eclipse Marketplace...)
-* File -> Import... -> Existing Maven Projects -> Select `simplelanguage` folder -> Finish
-
-### Netbeans
-* Tested with Netbeans 8.2
-* Open Netbeans
-* File -> Open Project -> Select `simplelanguage` folder -> Check "Open Required Projects:" -> Open Project
-
-### IntelliJ IDEA
-* Tested with IntelliJ IDEA 2016.3.4
-* Open IntelliJ IDEA
-* File -> New -> Project from existing Sources -> Select `simplelanguage` folder -> Check "Create module groups for multi-module Maven projects" -> Finish
-
-## Running
-
-* Execute `./sl language/tests/HelloWorld.sl` to run a simple language source file.
-* Execute `./sl -disassemble language/tests/SumPrint.sl` to see assembly code for Truffle compiled functions.
-
-## IGV
-
-* Download the Ideal Graph Visualizer (IGV) from
-  https://lafo.ssw.uni-linz.ac.at/pub/idealgraphvisualizer/
-* Unpack the downloaded `.zip` file
-* Execute `bin/idealgraphvsiualizer` to start IGV
-* Execute `./sl -dump language/tests/SumPrint.sl` to dump graphs to IGV.
-
-## Debugging
-
-* Execute `./sl -debug language/tests/HelloWorld.sl`.
-* Attach a Java remote debugger (like Eclipse) on port 8000.
-
-## Tested Compatibility
-
-Simple language is compatible to:
-
-* Truffle-Version: 0.33
-* GraalVM-Version: 0.33
-
-## Further information
-
-* [Truffle JavaDoc](http://www.graalvm.org/truffle/javadoc/)
-* [Truffle on Github](https://github.com/oracle/graal/tree/master/truffle)
-* [Graal on Github](https://github.com/oracle/graal/tree/master/compiler)
-* [Truffle Tutorials and Presentations](https://github.com/oracle/graal/blob/master/docs/Publications.md)
-* [Graal VM]( http://www.oracle.com/technetwork/oracle-labs/program-languages/overview) on the Oracle Technology Network
-* [Papers on Truffle](http://ssw.jku.at/Research/Projects/JVM/Truffle.html)
-* [Papers on Graal](http://ssw.jku.at/Research/Projects/JVM/Graal.html)
-
-## License
-
-The Truffle framework is licensed under the [GPL 2 with Classpath exception](http://openjdk.java.net/legal/gplv2+ce.html).
-The SimpleLanguage is licensed under the [Universal Permissive License (UPL)](http://opensource.org/licenses/UPL).
-
-
+For instructions on how to get started please refer to [http://www.graalvm.org/docs/graalvm-as-a-platform/implement-language/](http://www.graalvm.org/docs/graalvm-as-a-platform/implement-language/)
```

Comment:
use website instead of http://www.graalvm.org/docs/graalvm-as-a-platform/implement-language/